filescppgen.py

```diff
@@ -82,14 +88,14 @@ def usage():
      ("ome-common-cpp/%s/source/" +
       "ome-common-cpp-%s.zip") % (
           common_version, common_version)),
-    ("BF_SOURCE_TXZ", RELATIVE_PATH +
-     ("bio-formats/%s/artifacts/" +
-      "bioformats-dfsg-%s.tar.xz") % (
-          bf_version, bf_version)),
-    ("BF_SOURCE_ZIP", RELATIVE_PATH +
-     ("bio-formats/%s/artifacts/" +
-      "bioformats-dfsg-%s.zip") % (
-          bf_version, bf_version)),
+    ("MODEL_SOURCE_TXZ", RELATIVE_PATH +
+     ("ome-model/%s/artifacts/" +
```

Comment:
See https://ci.openmicroscopy.org/view/Release/job/OME-FILES-CPP-DEV-release-downloads/132/console, this should be `source` not `artifacts